CleanCodeJN.GenericApis.Sample/Program.cs

```diff
@@ -16,6 +16,7 @@
 });
 
 builder.Services.RegisterRepositoriesCommandsWithAutomaticMapping<MyDbContext>();
+builder.Services.AddApiImplementations();
```

Comment:
Why do you need this feature. You can inject dependencies by using [FromService] in the minimal API, like I showcased in the Custom Delete case in the documentation. Why do you need to inject something into the IAPI classes?